models/channel.coffee

```diff
@@ -23,6 +23,9 @@ module.exports = class Channel extends Block
   href: ->
     "/#{@get('user')?.slug}/#{@get('slug')}"
 
+  nMoreBlocks: (n) ->
+    @get('length') - n
```

Comment:
Tbh I don't really see the need for it anyway. It's a simple comparison in the template.